.github/workflows/schedule.yml

```diff
@@ -0,0 +1,39 @@
+name: Conformance test harness
+
+on:
+  schedule:
+    - cron: '23 5 * * *'
+  push:
+    branches:
+     - test/conformance
```

Comment:
Change?

Reply:
I kept this just so we can check if it gets broken in case a next PR updates the script. But we can also just add it back then again.

Reply:
Makes sense, let's keep.